orcestra/flightplan.py

```diff
@@ -7,12 +7,15 @@
 from typing import Optional, List, Dict
 from warnings import warn
 import datetime
+import json
 
+import cartopy.crs as ccrs
```

Comment:
Other functions in `flightplan.py` do lazy imports of cartopy. Probably that's not really reasonable, as likely there'll be no usecase of flightplan without cartopy, but I guess we should be consistent, so I'd suggest to either:
* removing the other lazy imports or
* make this use lazy